src/applications/gi-sandbox/components/profile/Academics.jsx

```diff
@@ -85,8 +85,8 @@ export default function Academics({ institution, onShowModal }) {
   );
 
   const educationDetails = (
-    <div>
-      <h3>Education details</h3>
+    <div className="small-screen-font">
```

Comment:
```suggestion
    <div className="small-screen:vads-u-font-size--sm">
```
from https://design.va.gov/utilities/font-size ?